Packs/GoogleSafeBrowsing/Integrations/GoogleSafeBrowsingV2/GoogleSafeBrowsingV2_test.py

```diff
@@ -0,0 +1,152 @@
+                                                   'threatType': 'MALWARE'}],
+                                 'www.test1.com': []}
+
+
+def test_handle_errors():
+    """
+    Given:
+        - Handle errors function to return understandable error massage
+
+    When:
+        - the response contain error
+
+    Then:
+        - validating that the error massage is as expected
+    """
+    from GoogleSafeBrowsingV2 import handle_errors
+
+    with pytest.raises(Exception) as e:
+        handle_errors({'StatusCode': 250})
+    assert str(e.value) == 'Failed to perform request, request status code: 250.'
+
+    with pytest.raises(Exception) as e:
+        handle_errors({'StatusCode': 204, 'Body': ''})
+    assert str(e.value) == 'No content received. Possible API rate limit reached.'
+
+    with pytest.raises(Exception) as e:
+        handle_errors({'Body': ''})
+    assert str(e.value) == 'No content received. Maybe you tried a private API?.'
+
+    with pytest.raises(Exception) as e:
+        handle_errors({'error': {'message': 'massage', 'code': 'code'}})
+    assert str(e.value) == 'Failed accessing Google Safe Browsing APIs. Error: massage. Error code: code'
+
+
+URL_RESPONSE = {
+    'matches': [
+        {'threatType': 'MALWARE', 'platformType': 'ANY_PLATFORM', 'threat': {'url': 'url1'}, 'cacheDuration': '300s',
+         'threatEntryType': 'URL'},
+        {'threatType': 'MALWARE', 'platformType': 'WINDOWS', 'threat': {'url': 'url1'}, 'cacheDuration': '300s',
+         'threatEntryType': 'URL'}]}
+
+URL_CONTENTS = [
+    {'cacheDuration': '300s',
+     'platformType': 'ANY_PLATFORM',
+     'threat': {'url': 'url1'},
+     'threatEntryType': 'URL',
+     'threatType': 'MALWARE'},
+    {'cacheDuration': '300s',
+     'platformType': 'WINDOWS',
+     'threat': {'url': 'url1'},
+     'threatEntryType': 'URL',
+     'threatType': 'MALWARE'}
+]
+
+
+def test_command_url(mocker):
+    """
+    Given:
+        - Got url to check for scores
+
+    When:
+        - After asked fot url command
+
+    Then:
+        - validating that the IOC score is as expected
+        - validating the the Reliability is as expected
+        - validating the the Contents is as expected
+    """
+    from GoogleSafeBrowsingV2 import url_command
+    client = create_client(base_url="https://safebrowsing.googleapis.com/v4/threatMatches:find")
+    mocker.patch.object(client, '_http_request', return_value=URL_RESPONSE)
+
+    url_command = url_command(client, {'url': ['url1', 'url2']}, CLIENT_BODY)
+
+    # validate score
+    output = url_command[0].to_context().get('EntryContext', {})
+    dbot_key = 'DBotScore(val.Indicator && val.Indicator == obj.Indicator &&' \
+               ' val.Vendor == obj.Vendor && val.Type == obj.Type)'
+    assert output.get(dbot_key, [])[0].get('Score') == 3
+    assert output.get(dbot_key, [])[0].get('Reliability') == DBotScoreReliability.A_PLUS
```

Comment:
how is the defaultvalue in the params `B - Usually reliable` but here `A+`?
let's have the UT alligned with the integration params.
also, even if urls are mocked, with is fine, writing benign.com/malivious.com is better than url1.com.

Reply:
done